.gitignore

```diff
@@ -13,3 +13,6 @@ nbproject
 # Project Specific
 node_modules/
 *.log
+
+.env.local
+.env.*.local
```

Comment:
Having removed the `.env` file and port configurability we can remove these again (but there is no harm either).